### ISSUE TITLE
Allow default setting to show/hide sidebar

### ISSUE THREAD

**Issue by eyeseast** [documentcloud/wordpress-documentcloud#31]
The plugin allows defaults for size, but the sidebar is another frequently used setting. (I always turn it off in posts, for example.) 

Any other knobs and dials that should be default-able?

**Comment by reefdog** [comment]
Arguably, all booleans could be. Here's the list of accepted ones:

- `responsive` (currently defaults `true`)
- `sidebar` (currently default `false`)
- `text` (currently defaults `true`)
- `pdf` (currently defaults `true`, although this is made irrelevant by `sidebar:false`)
- `notes`
- `zoom`
- `search`

**Comment by eyeseast** [comment]
I think it makes sense to include all of them. Might as well make this easy for people.

**Comment by reefdog** [comment]
:+1: You wanna take a crack at it?

I don't much care what the defaults for `text/pdf/sidebar` are (I just left the existing defaults in), but `responsive` should default true.

**Comment by eyeseast** [comment]
Yeah, can do. I was also going to take a shot at #25 this week or next.

**Comment by reefdog** [comment]
Awesome! I'm about to tag and release v0.4.1 with the VIP conformity and #35. #25 would justify a v0.5.0 release, IMO.
